site/docs/bindings/node-js.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 1
+sidebar_label: Node.js bindings
+description: Node.js bindings provide JavaScript/TypeScript API.
+---
+
+# OpenVINO™ GenAI Node.js bindings
+
+OpenVINO GenAI provides Node.js bindings that enable you to use generative AI pipelines in JavaScript and TypeScript applications.
+
+:::note API Coverage
+JavaScript/TypeScript bindings currently provide a subset of the full OpenVINO GenAI API available in C++ and Python. The focus is on core text generation (LLMPipeline) and text embedding (TextEmbeddingPipeline) functionality.
```

Comment:
```suggestion
Node.js bindings currently provide a subset of the full OpenVINO GenAI API available in C++ and Python. The focus is on core text generation (`LLMPipeline`) and text embedding (`TextEmbeddingPipeline`) functionality.
```

There are several naming patterns in docs for JS bindings - Node.js bindings, JavaScript/TypeScript, JavaScript (in code block language tabs).
I think we should choose one and keep the same unified pattern over all documents/pages.
I would prefer concise labels: "Node.js" (as it shows that it is for Node environment, not browser) or just "JavaScript" (but probably with adding a note in one place indicating execution in Node env only).

Reply:
I agree with you. JS and TS are very similar, and both could easily be converted to the other. The only advantage of TS is its types. In my opinion, this really helps and makes the code clearer.
Using TS, we demonstrate that our library supports types.

P.S. I am going to convert all of our samples to TS, which will match the documentation.